python/spacewalk/satellite_tools/reposync.py

```diff
@@ -1192,19 +1192,27 @@ def _populate_erratum(self, notice):
             updated_date = self._to_db_date(notice["updated"])
         else:
             updated_date = self._to_db_date(notice["issued"])
+
+        existing_packages = existing_errata["packages"] if existing_errata else []
+        npkgs = [pkg for c in notice["pkglist"] for pkg in c["packages"]]
```

Comment:
Please don't use list comprehensions with multiple for clauses ([style guide](https://google.github.io/styleguide/pyguide.html#27-comprehensions--generator-expressions)).